packages/astro/e2e/view-transitions.test.js

```diff
@@ -426,6 +423,85 @@ test.describe('View Transitions', () => {
 		await expect(locator).toBeInViewport();
 	});
 
+	test('View Transitions Rule', async ({ page, astro }) => {
+		let consoleCount = 0;
+		page.on('console', (msg) => {
+			// This count is used for transition events
+			if (msg.text() === 'ready') consoleCount++;
+		});
+		// Don't test back and forward '' to '', because They are not stored in the history.
+		// click '' to '' (transition)
+		await page.goto(astro.resolveUrl('/long-page'));
```

Comment:
needs some parameter (or a copy of the fixture) once the "feature toggle" is im place